modules/osdk-golang-create-deploy.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/golang/osdk-quickstart.adoc
+
+[id="osdk-golang-create-deploy_{context}"]
+= Creating and deploying a Go-based Operator
+
+Operator authors can follow this procedure to walk through an example for building and deploying a simple Memcached Operator.
+
+.Prerequisites
+
+- Operator SDK CLI installed on a development workstation
+- OpenShift CLI (`oc`) v{product-version}+ installed
+- Logged into a {product-title} {product-version} cluster with `oc` using an account with `cluster-admin` permissions
+
+.Procedure
+
+. *Create a project.*
+
+.. Create and change into a directory for your project:
++
+[source,terminal]
+----
+$ mkdir memcached-operator
+----
++
+[source,terminal]
+----
+$ cd memcached-operator
+----
+
+.. Run the `operator-sdk init` command with the Go plug-in to initialize the project:
```

Comment:
Please rewrite this step. I can't tell if this means that you're invoking the Go plug-in or if the Go plug-in is a hidden prereq or some other thing.